src/execution/compiler/operator/index_create_translator.cpp

```diff
@@ -36,71 +35,109 @@ IndexCreateTranslator::IndexCreateTranslator(const planner::CreateIndexPlanNode
     compilation_context->Prepare(*index_col.StoredExpression());
   }
   pipeline->RegisterSource(this, Pipeline::Parallelism::Serial);
```

Comment:
Why not parallel by default? Had to change this to test.

Reply:
My bad. I forgot to change it back to parallel, as I was also testing serial mode after modification